src/test/java/apicurito/tests/steps/ConfigurationOCPSteps.java

```diff
@@ -146,57 +153,36 @@ public void checkThatApicuritoOperatorIsDeployedAndInRunningState() {
-                quayUser);
-            index.addBundle(ReleaseSpecificParameters.APICURITO_OPERATOR_7_8_METADATA_URL);
-            index.addBundle(ReleaseSpecificParameters.APICURITO_OPERATOR_7_9_METADATA_URL);
-            index.addBundle(ReleaseSpecificParameters.APICURITO_OPERATOR_7_10_0_METADATA_URL);
-            index.addBundle(ReleaseSpecificParameters.APICURITO_OPERATOR_7_10_1_METADATA_URL);
-
-            oldBundle = index.addBundle(ReleaseSpecificParameters.APICURITO_OPERATOR_7_10_2_METADATA_URL);
-            currentBundle = index.addBundle(TestConfiguration.apicuritoOperatorMetadataUrl());
-            try {
-                index.addIndexToCluster("apicurito-test-catalog");
-            } catch (InterruptedException | TimeoutException | IOException e) {
-                log.error("Adding the index to cluster failed", e);
-            }
-        }
-    }
-
-    @When("deploy operator from operatorhub")
-    public void deployOperatorHub() {
-        try {
-            log.info("Creating apicurito subscription");
-            currentBundle.createSubscription();
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-    }
-
     @When("deploy older operator from operatorhub")
     public void deployOlderOperatorHub() {
-        try {
-            log.info("Creating apicurito subscription");
-            oldBundle.createSubscription();
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
+        log.info("Creating apicurito subscription");
+        ApicuritoOperatorhub.deploy(ReleaseSpecificParameters.APICURITO_OLD_UPDATE_CHANNEL, "older");
     }
 
     @When("upgrade operator via operatorhub")
     public void upgradeOperatorhub() {
         log.info("Upgrading apicurito subscription");
-        oldBundle.update(currentBundle, true);
-        ConfigurationOCPUtils.waitForOperatorUpdate(2);
+
+        try {
+            OpenShift ocp = ConfigurationOCPSteps.getOpenShiftService().getClient();
+            String namespace = ConfigurationOCPSteps.getOpenShiftService().getClient().getNamespace();
+            JSONObject subscription = ApicuritoOperatorhub.getSubscription();
+            assertThat(subscription).describedAs("Apicurito subscription was not found in " + namespace + " namespace.").isNotNull();
+            subscription.getJSONObject("spec").put("channel", ReleaseSpecificParameters.APICURITO_CURRENT_UPDATE_CHANNEL);
+            HelperFunctions.waitFor(() -> {
+                try {
+                    ocp.customResource(ApicuritoOperatorhub.getSubscriptionContext()).edit(namespace, SUBSCRIPTION_NAME, subscription.toString());
+                    return true;
+                } catch (Exception e) {
+                    return false;
+                }
+            }, 300L, 120L);
+            ConfigurationOCPUtils.waitForOperatorUpdate();
+            ConfigurationOCPUtils.waitForPodsUpdate(2);
+        } catch (Exception e) {
```

Comment:
waitFor method throws these two exceptions: `InterruptedException` `TimeoutException`. Please catch them instead of `Exception`